### PR TITLE
node tutorial update

### PR DESCRIPTION
**Pull request by gabriel-okteto**

### CODE REVIEW

File: src/content/samples/node.mdx [hunk 1]
```diff
@@ -25,11 +25,29 @@ $ git clone https://github.com/okteto/node-getting-started
...
+```
+build:
+  hello-world:
+    image: okteto.dev/node-hello-world:1.0.0
+    context: .
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto/dev:latest
```
Comment (pchico83) [inline src/content/samples/node.mdx:39]: remove `image`

File: src/content/samples/node.mdx [hunk 2]
```diff
@@ -25,11 +25,29 @@ $ git clone https://github.com/okteto/node-getting-started
...
+```
+
+Deploy your development environment by executing:
 
 ```console
-$ kubectl apply -f k8s.yml
+$ okteto deploy --build
 ```
 
 ```console
```
Comment (pchico83) [inline src/content/samples/node.mdx:53]: update the console output:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16R47

File: src/content/samples/node.mdx [hunk 3]
```diff
@@ -43,58 +61,7 @@ Log into [Okteto Cloud](https://cloud.okteto.com/#/?origin=docs) and click on th
...
-- `name`: the name of the Kubernetes deployment you want to put on development mode
-- `command`: the start command of the development container
-- `sync`: the folders that will be synchronized between your local machine and the development container
-- `forward`: a list of ports to forward from your development container
-
-Also, the `okteto init` command creates a `.stignore` file to indicate which files shouldn't be synchronized to your development container.
-This is useful to avoid synchronizing binaries, build artifacts, git metadata, or dependencies like `node_modules`.
-
-## Step 3: Activate your development container
+## Step 2: Activate your development container
```
Comment (pchico83) [inline src/content/samples/node.mdx:76]: You are missing changes in this section:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16R68

File: src/content/samples/node.mdx [hunk 4]
```diff
@@ -135,7 +102,7 @@ Starting hello-world server...
 
 Go back to the browser and reload the page to test that your application is running.
 
-## Step 4: Develop directly on Okteto Cloud
+## Step 3: Develop directly on Okteto Cloud
```
Comment (pchico83) [inline src/content/samples/node.mdx:117]: Update the output in this section:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16R107

File: src/content/samples/node.mdx [hunk 5]
```diff
@@ -154,7 +121,7 @@ Starting hello-world server...
 
 Go back to the browser and reload the page. Your code changes were instantly applied. No commit, build, or push required 😎!
 
-## Step 5: Debug directly on Okteto Cloud
+## Step 4: Debug directly on Okteto Cloud
```
Comment (pchico83) [inline src/content/samples/node.mdx:136]: You are missing this change:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16L225

File: src/content/samples/node.mdx (hunk 2 above)
Comment (gabriel-okteto) [reply src/content/samples/node.mdx:53]: all of it?

Comment (pchico83) [reply src/content/samples/node.mdx:53]: What I usually do is follow the sample and add the output. In the golang sample, I used `...` for the build output, you can check it in this PR:
https://github.com/okteto/docs/pull/119

File: src/content/samples/node.mdx (hunk 1 above)
Comment (pchico83) [reply src/content/samples/node.mdx:39]: the `image` field is still there, no?

File: src/content/samples/node.mdx [hunk 6]
```diff
@@ -25,76 +25,73 @@ $ git clone https://github.com/okteto/node-getting-started
...
-- `name`: the name of the Kubernetes deployment you want to put on development mode
-- `command`: the start command of the development container
-- `sync`: the folders that will be synchronized between your local machine and the development container
-- `forward`: a list of ports to forward from your development container
+## Step 2: Activate your development container
+The [dev](/docs/reference/manifest/#dev-object-optional) section defines how to activate a development container for the Node Sample App:
+```
+dev:
+  hello-world:
+    image: okteto/dev:latest
```
Comment (pchico83) [inline src/content/samples/node.mdx:79]: remove `image` field

File: src/content/samples/node.mdx [hunk 7]
```diff
@@ -25,76 +25,73 @@ $ git clone https://github.com/okteto/node-getting-started
...
+    command: bash
+    sync:
+      - .:/usr/src/app
+    forward:
+      - 9229:9229
+```
+The `hello-world` key matches the name of the hello world Deployment. The meaning of the rest of fields is:
+- `image`: the image used by the development container. More information on development images [here](/docs/reference/development-environments/).
```
Comment (pchico83) [inline src/content/samples/node.mdx:87]: remove `image` field

File: src/content/samples/node.mdx [hunk 8]
```diff
@@ -25,76 +25,73 @@ $ git clone https://github.com/okteto/node-getting-started
...
+- `command`: the start command of the development container.
+- `sync`: the folders that will be synchronized between your local machine and the development container.
+- `forward`: a list of ports to forward from your development container to locahost in your machine. This is needed to configure the Go debugger.
```
Comment (pchico83) [inline src/content/samples/node.mdx:90]:
```suggestion
- `forward`: a list of ports to forward from your development container to locahost in your machine. This is needed to configure the Nodejs debugger.
```

File: src/content/samples/node.mdx [hunk 9]
```diff
@@ -115,16 +112,17 @@ Welcome to your development container. Happy coding!
 cindy:hello-world app>
 ```
 
+> - Test your application end to end as fast as you type code
```
Comment (pchico83) [inline src/content/samples/node.mdx:115]: why these changes here?

File: src/content/samples/node.mdx [hunk 10]
```diff
@@ -115,16 +112,17 @@ Welcome to your development container. Happy coding!
...
+> - No more CPU cycles wasted in your machine. Develop at the speed of the cloud!
+
 Working in your development container is the same as working on your local machine.
 Start the application in hot-reload mode by running the following command:
 
 ```console
-cindy:hello-world app> nodemon index.js
+cindy:hello-world app> npm run start
```
Comment (pchico83) [inline src/content/samples/node.mdx:122]: why is this change?

File: src/content/samples/node.mdx (hunk 9 above)
Comment (gabriel-okteto) [reply src/content/samples/node.mdx:115]: Consistently, it was in the golang docs that you added L213

File: src/content/samples/node.mdx [hunk 11]
```diff
@@ -25,76 +25,73 @@ $ git clone https://github.com/okteto/node-getting-started
...
-```console
-deployment.apps "hello-world" created
-service "hello-world" created
+build:
+  hello-world:
+    image: okteto.dev/node-hello-world:1.0.0
+    context: .
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
```
Comment (pchico83) [inline src/content/samples/node.mdx:37]: Since we are explaining the `dev` section later (and adding the `dev` section manifest later`, I would skip the `dev` section at this point

File: src/content/samples/node.mdx [hunk 12]
```diff
@@ -25,76 +25,65 @@ $ git clone https://github.com/okteto/node-getting-started
...
```
Comment (pchico83) [inline src/content/samples/node.mdx:37]: Let's don't show the `dev` section as we do here:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16R38

File: src/content/samples/node.mdx (hunk 9 above)
Comment (pchico83) [reply src/content/samples/node.mdx:107]: But not here. They are in the `Next steps` section

File: src/content/samples/node.mdx (hunk 10 above)
Comment (pchico83) [reply src/content/samples/node.mdx:114]: sounds good

**Review by rberrelleza — APPROVED** [review]

File: src/content/samples/node.mdx [hunk 13]
```diff
@@ -13,7 +13,7 @@ This tutorial will show you how to develop and debug a Node.js sample applicatio
 
 ## Prerequisites
 
-- Install the Okteto CLI (>= 1.12.13). Follow this [guide](/docs/cloud/okteto-cli/) if you haven't done it yet.
+- Install the Okteto CLI (>= 2.0.0). Follow this [guide](/docs/cloud/okteto-cli/) if you haven't done it yet.
```
Comment (pchico83) [inline src/content/samples/node.mdx:16]: Can we apply this change too?
https://github.com/okteto/docs/pull/124

**Review by pchico83 — APPROVED** [review]
